trade-finance-manager-ui/server/types/view-models/utilisation-report-reconciliation-for-report-view-model.ts

```diff
@@ -107,4 +120,7 @@ export type UtilisationReportReconciliationForReportViewModel = BaseViewModel &
   enablePaymentsReceivedSorting: boolean;
   keyingSheet: KeyingSheetViewModel;
   paymentDetails: PaymentDetailsViewModel;
+  paymentDetailsFilters?: PaymentDetailsFiltersViewModel;
+  paymentDetailsFilterErrors: PaymentDetailsFilterErrorsViewModel;
+  isPaymentDetailsFilterActive: boolean;
```

Comment:
Should these be in `PaymentDetailsViewModel`, since they seem to be part of the "payment details view/form?

Also should `PaymentDetailsFilterErrorsViewModel` be optional or will there always be an error?

Reply:
Thanks, good points, I've made both these changes

All these additional props now live inside `PaymentDetailsViewModel` (passed in to the view model as `paymentDetails`)